bai2.js

```diff
@@ -0,0 +1,9 @@
+function sumArray(arr) {
+  var sum = 0;
+  var arrLength = arr.length;
+  for (var i = 0; i < arrLength; i++) {
```

Comment:
```suggestion
  for (let i = 0; i < arrLength; i++) {
```
`The difference between let and var is in the scope of the variables they create: Variables declared by let are only available inside the block where they're defined. Variables declared by var are available throughout the function in which they're declared.21 thg 10, 2022
`